onboard/nucleo_bridge/README.md

```diff
@@ -0,0 +1,112 @@
+Subscriber: onboard/nucleo_bridge
+
+#### Foot Open Loop \[Subscriber\] "/foot_openloop_cmd"
+Message: [FootCmd.lcm](https://github.com/raytitan/mrover-workspace/blob/rnucleo/rover_msgs/FootCmd.lcm) \
+Publisher: onboard/teleop \
+Subscriber: onboard/nucleo_bridge
+
+#### RA Pos Data \[Publisher\] "/arm_posiiton"
+Message: [ArmPosition.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/ArmPosition.lcm) \
+Publisher: onboard/nucleo_bridge \
+Subscriber: onboard/kinematics
+
+#### SA Pos Data \[Publisher\] "/sa_pos_data"
+Message: [SAPosData.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/SAPosData.lcm) \
+Publisher: onboard/nucleo_bridge \
+Subscriber: onboard/kinematics
+
+### Usage
+
+To build nucleo_bridge use `$./jarvis build onboard/nucleo_bridge/ ` from the mrover-workspace directory.
+
+To run nucleo_bridge use `$./jarvis exec onboard/nucleo_bridge/ `
+from the mrover-workspace directory.
+
+After initializing the LCM bus, I2C bus, and virtual Controller objects, the CLI will only show errors, since printing output to the console is time expensive. A blank CLI is a good thing.
+
+To control the RA/SA through open-loop
+* Ensure onboard/teleop is running on the same platform
+* Ensure base_station/gui is running on the base station
+* Operate the joysticks with the RA/SA Task page open on the base station GUI
+
+To control the RA/SA through closed-loop
+* Ensure onboard/teleop is running on the same platform
+* Ensure onboard/kinematics is running on the same platform
+* Ensure base_station/kineval is running on the base station
+* Input commands into the arm control page on the base station GUI
+
+### Common Errors
+
+This routine typically only thows one type of error, when it has issues communicating with the Nucleo motor controllers. They will have the form "command failed on channel"
+
+The Nucleo that is unresponsive is the first digit of the channel minus 1. For example, if the error message says "activate failed on 10", check Nucleo 0. Typically this is because the wiring on the i2c bus has failed nd needs to be fixed. Even after wiring is fixed or other related issues have been resolved, the Nucleos,To reset the Nucleo, press it's power button. If this does not fix nucleo_bridge, you may have to restart nucleo_bridge.
+
+These communication errors can be caused by a failure anywhere on the i2c bus, so it is not unlikely that issues with only one Nucleo will cause the all the Nucleos to fail to communicate properly. 
+
+### Notes
+
+To reference the firmware that is loaded onto the Nucleos, [see this] (https://github.com/raytitan/NucleoHAL/tree/master/Core). A README for the firmware is coming soon, which will point out details such as watchdog timers and common issues.
+
+### Todo
+
+- [ ] Re-enable disabled arm joints and lcm messages.
+- [ ] Hardcode the encoder configurations, PID constants, and other miscellanious variables for each arm joint.
+- [ ] Test predictive torques.

```

Comment:
Make issues out of these and remove this section

Reply:
Implemented. See #540. Todos changed based off of fall semester progress.